salt/freifunk/base/ddmesh/var/www_freifunk/sysinfo.json

```diff
@@ -94,16 +107,27 @@ function combined_statistics() {
 			fi
 		done
 
+	stat_backbone_mesh_rx=$((stat_backbone_mesh_wg_rx + stat_backbone_mesh_fastd_rx))
+	stat_backbone_mesh_tx=$((stat_backbone_mesh_wg_tx + stat_backbone_mesh_fastd_tx))
+
 	# output
 	cat << EOM
 			"wan_rx":"${stat_wan_rx}",
 			"wan_tx":"${stat_wan_tx}",
 			"backbone_mesh_rx":"${stat_backbone_mesh_rx}",
 			"backbone_mesh_tx":"${stat_backbone_mesh_tx}",
-			"gateway_rx":"${stat_gateway_rx}",
-			"gateway_tx":"${stat_gateway_tx}",
 			"vpn_rx":"${stat_vpn_rx}",
-			"vpn_tx":"${stat_vpn_tx}"
+			"vpn_tx":"${stat_vpn_tx}",
+			"details": {
+				"wifi_mesh_2g_rx":"0",
+				"wifi_mesh_2g_tx":"0",
+				"wifi_mesh_5g_rx":"0",
+				"wifi_mesh_5g_tx":"0",
```

Comment:
Die Werte die mit "0" in der Sysinfo stehen können auch raus, damit kann ich umgehen, wenn Interfaces nicht da sind.

Reply:
habe ich gemacht. ist auch sauberer, da der server ja kein wifi hat.